### ISSUE TITLE
Fix user fixtures and test setup

### ISSUE THREAD

**Issue by samarpan1738** [Real-Dev-Squad/website-backend#659]
**Problem**
In real life scenario when a user is first added in the DB they do not have any roles.

But the [fixtures](https://github.com/Real-Dev-Squad/website-backend/blob/develop/test/fixtures/user/user.js) being used to add a new user (during the tests setup) has the roles property pre-populated and we end up with a scenario where a new user added in the DB already has roles. 

Hence we need to fix the fixtures and modify the test setup part where the new user added in DB must not have any roles. 
If we want to add roles it must be done in the same way as it'll be done in a real life scenario

Basically the tests should mimic the business flow and use cases 

**How we found this issue?**
Read this discussion : https://github.com/Real-Dev-Squad/website-backend/pull/654#discussion_r939069800
